src/iconsManifest/supportedFolders.ts

```diff
@@ -105,6 +105,19 @@ export const extensions: IFolderCollection = {
     { icon: 'github', extensions: ['.github'], format: FileFormat.svg },
     { icon: 'gitlab', extensions: ['.gitlab'], format: FileFormat.svg },
     { icon: 'graphql', extensions: ['graphql'], format: FileFormat.svg },
+    {
+      icon: 'gulp',
+      extensions: [
+        'gulp',
+        'gulpfile.js',
+        'gulpfile.coffee',
+        'gulpfile.ts',
+        'gulpfile.babel.js',
+        'gulpfile.babel.coffee',
+        'gulpfile.babel.ts',
```

Comment:
This means a folder is called `gulpfile.babel.*`?

Reply:
Yes. See the comment https://github.com/vscode-icons/vscode-icons/issues/650#issuecomment-450661168.

Reply:
Oh wow. Thanks for the clarification.